lib/server_health_check.rb

```diff
@@ -25,13 +25,17 @@ def redis!(host: nil, port: 6379)
 
   def active_record!
     begin
-      ActiveRecord::Base.connection
-      @results[:active_record] = OK
-      true
+      if ActiveRecord::Base.connection_pool.with_connection { |connection| connection.active? }
+        @results[:active_record] = OK
+        true
+      else
+        @results[:active_record] = "Failed: unable to connect to database"
+        false
+      end
     rescue NameError
```

Comment:
I concur that raising `NameError` is the correct thing to do, since to execute this code, you have to explicitly specify that you want the ActiveRecord check, as per the readme:

```ruby
    health_check = ServerHealthCheck.new
    health_check.active_record!
```